asdf/tagged.py

```diff
@@ -35,9 +35,9 @@
 
 import six
 
+from astropy import time
```

Comment:
There's a not-quite-finished pull requested @mdboom started in #160 to make Astropy not a strict dependency of asdf.  It will probably take a bit of work to revive that PR but it would probably be for the best, and this would be at odds with that.

I'm probably missing some context to this PR though, but it should be handled in such a way that doesn't make Astropy a requirement.  (This goes back to my earlier comment about there needing to be an API that *astropy*, when imported, could register additional `Tagged` types...)